services/QuillLMS/app/workers/snapshots/cache_snapshot_previous_count_worker.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
This change is from https://github.com/empirical-org/Empirical-Core/pull/11138 and does not need to be reviewed here.